queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+SELECT name FROM animals WHERE CAST(to_char(date_of_birth, 'YYYY') AS INTEGER) BETWEEN 2016 AND 2019;
+
+SELECT name FROM animals WHERE  neutered IS TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name LIKE 'Agumon' OR name LIKE 'Pikachu';
+
```

Comment:
- [ ] The `LIKE` operator is used to match a pattern meaning that this query will still return the date_of_birth of the animal with names `Agumonu` and `Pikachua`. Kindly optimise this query by using a more specific query. hint; 

```sql
SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';

```

Reply:
Done